# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/audioipc/client/src/stream.rs
# Commit: 0a7873a6b522
# Full Hash: 0a7873a6b522ce4aef9a32ae4a47f093ba33e0f4
# Author: Matthew Gregan <kinetik@flim.org>
# Date: 2020-01-26 09:48:16
# Regressor Bug: 1590249
# File Overlap Count: 3
# Description:
#   Bug 1590249 - Update audioipc to 86d49ddf.  r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D60615
# ==============================================================================

diff -r 36f6505b50c9 -r 0a7873a6b522 media/audioipc/client/src/stream.rs
--- a/media/audioipc/client/src/stream.rs	Sat Jan 25 21:30:14 2020 +0000
+++ b/media/audioipc/client/src/stream.rs	Sat Jan 25 21:31:32 2020 +0000
@@ -178,19 +178,16 @@
             data.token, data.platform_handles
         );
 
-        let stm = data.platform_handles[0];
-        let stream = unsafe { audioipc::MessageStream::from_raw_fd(stm.as_raw()) };
+        let stream = unsafe { audioipc::MessageStream::from_raw_fd(data.platform_handles[0].into_raw()) };
 
-        let input = data.platform_handles[1];
-        let input_file = unsafe { input.into_file() };
+        let input_file = unsafe { data.platform_handles[1].into_file() };
         let input_shm = if has_input {
             Some(SharedMemSlice::from(&input_file, audioipc::SHM_AREA_SIZE).unwrap())
         } else {
             None
         };
 
-        let output = data.platform_handles[2];
-        let output_file = unsafe { output.into_file() };
+        let output_file = unsafe { data.platform_handles[2].into_file() };
         let output_shm = if has_output {
             Some(SharedMemMutSlice::from(&output_file, audioipc::SHM_AREA_SIZE).unwrap())
         } else {